internal/resolution/variablesources/bundle_test.go

```diff
@@ -4,19 +4,344 @@ import (
 	"context"
 	"encoding/json"
 	"errors"
+	"testing"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	"github.com/operator-framework/deppy/pkg/deppy"
 	"github.com/operator-framework/operator-registry/alpha/declcfg"
 	"github.com/operator-framework/operator-registry/alpha/property"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 
 	"github.com/operator-framework/operator-controller/internal/catalogmetadata"
 	olmvariables "github.com/operator-framework/operator-controller/internal/resolution/variables"
 	"github.com/operator-framework/operator-controller/internal/resolution/variablesources"
 	testutil "github.com/operator-framework/operator-controller/test/util"
 )
 
+func TestMakeBundleVariables(t *testing.T) {
+	channel := catalogmetadata.Channel{Channel: declcfg.Channel{Name: "stable"}}
+	allBundles := []*catalogmetadata.Bundle{
```

Comment:
Similar to part one: https://github.com/operator-framework/operator-controller/pull/497#discussion_r1377929603

Need to check whether we can reduce the size of the test data.

Reply:
Right, I think we can get rid of a bunch of bundles here as many of them also seem to be related to GVK requirements which we do not care about here.

Reply:
Got rid of more than half of test data